search-resources-api/src/test/java/no/unit/nva/search/SearchResourcesApiHandlerTest.java

```diff
@@ -43,12 +44,11 @@ public void defaultConstructorThrowsIllegalStateExceptionWhenEnvironmentNotDefin
     }
 
     @Test
-    public void processInputReturnsNullWhenInputIsEmpty() throws ApiGatewayException {
+    public void processInputReturnsNullWhenInputIsEmpty() {
         RequestInfo requestInfo = mock(RequestInfo.class);
         Context context = mock(Context.class);
-        assertThrows(ApiGatewayException.class, () ->  searchResourcesApiHandler.processInput(null,
-                requestInfo,
-                context));
+        Executable executable = () -> searchResourcesApiHandler.processInput(null, requestInfo, context);
+        assertThrows(ApiGatewayException.class, executable);
```

Comment:
Very generic test that can succeed for other reasons than that expected here. SearchException is thrown at the moment. Consider deleting the test if this case is covered elsewhere